docs/azure/sdk/authentication/local-development-dev-accounts.md

```diff
@@ -1,135 +1,40 @@
 ---
 title: Authenticate .NET apps to Azure using developer accounts
-description: Learn how to authenticate your application to Azure services when using the Azure SDK for .NET during local development using developer accounts.
+description: Learn how to authenticate your application to Azure services when using the Azure SDK for .NET during local development using user accounts.
```

Comment:
Stick with "developer". I think that does a better job of communicating the scenario to customers.